### DISCUSSION TITLE
Standalone HTTPS mode ssl key error

### DISCUSSION THREAD

**Discussion by HaniCera** [cantaloupe-project/cantaloupe#517] [Q&A]
Hello, I'm trying to implement cantaloupe 4.1.5 on my spring boot test server via HTTPS protocol.

On cantaloupe manual 

"In standalone mode, Cantaloupe supports TLS connections over HTTPS, configurable via the https.* keys in the configuration file. The general process for getting this working is to add a signed X.509 certificate to either a Java KeyStore (JKS) or PKCS#12 key store, and then refer to the key store file with the https.key_store_path configuration option."

So I add my PKCS12 key and password (which works on my spring boot server, and it works well.) in properties file.

![properties](https://user-images.githubusercontent.com/42179682/124443068-54b80880-ddb8-11eb-8a82-e7910890ba7b.jpg)

but, when I turn on image server with standalone mode, it keeps return error log like below.

![log](https://user-images.githubusercontent.com/42179682/124443114-600b3400-ddb8-11eb-9936-75574664f1d5.jpg)

I bought my SSL certification from Sectigo and it works well on actual server with domain via HTTPS protocol.

what should I do with my sectigo sertification files for cantaloupe server?

**Comment by adolski** [comment]
Hi @HaniCera,

I believe the basic issue here is described in #395, which was fixed in 4.1.7.
